goss-testing/tests/common/goss-interface-has-ip.yaml

```diff
@@ -29,10 +29,20 @@ command:
     meta:
       desc: Validates that interface '{{$interface}}' has an IP address
       sev: 0
-    exec: "ip addr show dev {{$interface}} scope global | grep 'inet ' | awk '{print $2}'"
+    # For bond0.can0, if it does not exist, we do not fail. Echo a fake IP so the test passes
+    exec: |-
+      if ip link show dev {{$interface}}; then
+        ip -4 a s dev {{$interface}} | awk '/inet / {print $2}'
+      elif [[ {{$interface}} == "bond0.can0" ]]; then
```

Comment:
You should remove this entire "elif" clause now that you added the conditional around the skip statement

Reply:
Fixed